user_docs/en/userGuide.md

```diff
@@ -3333,6 +3333,18 @@ Some GDI applications will highlight text with a background color, NVDA (via dis
 In some situations, the text background may be entirely transparent, with the text layered on some other GUI element.
 With several historically popular GUI APIs, the text may be rendered with a transparent background, but visually the background color is accurate.
 
+##### Prevent system lock during say all or reading with Braille {#preventSystemLock}
+
+| . {.hideHeaderRow} |.|
+|---|---|
+|Options |Default (Enabled), Disabled, Enabled|
+|Default |Disabled|
+
+This option ensures that the screen stays on when reading with say all or with braille (e.g. when pressing scroll buttons).
+This avoids the situation where the screen unexpectedly locks during a say all.
```

Comment:
The option name talks about system lock whereas here, you talk about screen lock. I have not reviewed this PR in detail, but reading the UG does not clarify things for me.

Are we talking about system lock, I guess session lock?) or the screen turning off (black)?

I think that you should describe both the use cases for enabled and for disabled here. Thanks.

Reply:
More specifically and reading https://github.com/nvaccess/nvda/pull/17651#issuecomment-2623692235 by @btman16, I have concerns regarding braille.
I'd say that systems using modern standby should never enter standby, thus never lock, when reading braille.